### PR TITLE
Fix menu items not showing on mobile

### LINKED ISSUE #29004: BO - Menu - The Title of the subtab AdminParent are not displayed when the menu is resized
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md)
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add screenshots

The Title of the subtab AdminParent is not displayed when the menu is resized
image

I attached a screen record

https://user-images.githubusercontent.com/16067358/177953628-15dbe85d-e0ad-4cfa-9527-260dc27d34fb.mp4

PS1778 => ok

https://user-images.githubusercontent.com/16067358/177953888-c787cce1-253e-44f0-8e9d-ddbad160a720.mp4

### Expected behavior

It should be
image

### Steps to reproduce

1. Using the desktop > Go to BO 
2. Reduce the menu 
image
3. Resize the window to 900px * 100px
4. Click on the mobile menu 
image
5. See error 
image
6. If you are in a migrated page, try to click on the "keyboard_arrow_down" 
image
7. See error: the menu cannot be displayed

### PrestaShop version(s) where the bug happened

1.7.8.6, 8.0.x

### PHP version(s) where the bug happened

7.2

### If your bug is related to a module, specify its name and its version

_No response_

### PR DESCRIPTION
**Pull request by NeOMakinG**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 1.7.8.x
| Description?      | Menu items were not showing on mobile when you close the sidebar on desktop and then resize the window
| Type?             | bug fix
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fixes #29004.
| How to test?      | See issue
| Possible impacts? | BO Menu

### CODE REVIEW

File: admin-dev/themes/new-theme/js/components/navbar-transition-handler.js [hunk 1]
```diff
@@ -47,7 +50,10 @@ function NavbarTransitionHandler($navBar, $mainMenu, endTransitionEvent, $body)
 
     this.$navBar.removeEventListener(this.endTransitionEvent, this.showNavBarContent);
     const isSidebarClosed = this.$body.hasClass('page-sidebar-closed');
-    this.$mainMenu.toggleClass('sidebar-closed', isSidebarClosed);
+
+    if ($(window).width() > MAX_MOBILE_WIDTH) {
+      this.$mainMenu.toggleClass('sidebar-closed', isSidebarClosed);
```
Comment (kpodemski) [inline admin-dev/themes/new-theme/js/components/navbar-transition-handler.js:55]: I'm a bit picky, but don't you think that we should use a class map for every edited component in the new theme?

Comment (NeOMakinG) [reply admin-dev/themes/new-theme/js/components/navbar-transition-handler.js:55]: I guess it doesn't have too much sense if it's not a real selector, also, class maps are not used by anyone, really nitpick to me

**Review by kpodemski — APPROVED** [review]

**Review by atomiix — APPROVED** [review]

**Comment by sallemiines** [comment]
i will check this PR

**Comment by paulnoelcholot** [comment]
I will check this pr

**Comment by paulnoelcholot** [comment]
Hello @NeOMakinG !

- Automatic tests == OK
- Manuel tests == OK

Step to reproduice : 

Using the desktop > Go to BO
Reduce the menu
Resize the window to mobile screen
Click on the mobile menu
Check the menu dropdowns

It's OK for me

https://watch.screencastify.com/v/k9BzdawxONpFc826quOT

@PrestaShop/qa-functional

**Pull request merged**